volume/local/local.go

```diff
@@ -228,7 +230,7 @@ func (r *Root) Remove(v volume.Volume) error {
 	}
 
 	if err := removePath(realPath); err != nil {
-		return err
+		return errors.Wrap(err, fmt.Sprintf("error removing volume path '%s'", realPath))
```

Comment:
Usually, the rule is that in-pkg errors shouldn't be wrapped. This may make more sense inside `removePath`. Same with `v.Mount()`.